css/styles.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Verdana, sans-serif;
```

Comment:
We don't use Verdana in this project. The two fonts we use are Open Sans, and Proza Libre